apps/meteor/server/routes/avatar/middlewares/auth.js

```diff
@@ -3,8 +3,8 @@ import { Meteor } from 'meteor/meteor';
 import { userCanAccessAvatar } from '../utils';
 
 // protect all avatar endpoints
-export const protectAvatars = Meteor.bindEnvironment((req, res, next) => {
-	if (!userCanAccessAvatar(req)) {
+export const protectAvatars = Meteor.bindEnvironment(async (req, res, next) => {
```

Comment:
Is the place using this function awaiting it?

Reply:
It's being used inside the router handler, I don't think we need to await for the function there.